server/api/login.js

```diff
@@ -0,0 +1,51 @@
+const Crypto = require("crypto");
+import axios from "axios";
+
+var db = require("../db");
```

Comment:
use `const`, and a merge from main will break this slightly as you'll need `{ db }` instead of just `db`